base/compiler/optimize.jl

```diff
@@ -112,13 +112,21 @@ struct InliningState{Interp<:AbstractInterpreter}
     et::Union{EdgeTracker,Nothing}
     world::UInt
     interp::Interp
+    function InliningState(linfo::MethodInstance, params::OptimizationParams,
```

Comment:
New code probably shouldn't use the old linfo::LambdaInfo naming scheme, now that's been renamed to mi::MethodInstance?